app/jobs/send_emails_after_three_weeks_job.rb

```diff
@@ -0,0 +1,20 @@
+# Runs every morning at 4am. Finds claims which are not yet checked and which
+# are three weeks old on the date specified (defaulting to today), and sends an
+# email saying we are still in the process of checking.
+
+class SendEmailsAfterThreeWeeksJob < CronJob
+  self.cron_expression = "0 4 * * *"
+
+  def perform
+    Rails.logger.info "Sending three-week emails to #{three_week_old_unchecked_claims.count} unsubmitted claims from the database"
+    three_week_old_unchecked_claims.each do |claim|
+      ClaimMailer.update_after_three_weeks(claim).deliver_later
+    end
+  end
+
+  # private
+
+  def three_week_old_unchecked_claims
+    Claim.awaiting_checking.select { |claim| claim.days_since_submission == 21 }
```

Comment:
You can probably do this via ActiveRecord, rather than filtering through all the claims. Something like:

```
Claim.awaiting_checking.where(submitted_at: (21.days.ago.beginning_of_day...21.days.ago.end_of_day))
```

This generates the following SQL:

```
"SELECT "claims".* FROM "claims" LEFT OUTER JOIN "checks" ON "checks"."claim_id" = "claims"."id" WHERE "claims"."submitted_at" IS NOT NULL AND "checks"."claim_id" IS NULL AND "claims"."submitted_at" >= '2020-01-06 00:00:00' AND "claims"."submitted_at" < '2020-01-06 23:59:59.999999'"
```